README.md

```diff
@@ -41,7 +44,12 @@
 - **[Users can create a category.]**
 - **[Limit access to web app resources based on authorization rules.]**
 
-- Demo Link [Live Demo Link]()
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-app-2021.herokuapp.com/)
+
```

Comment:
- [ ] Kindly add a working link for the deployed app as requested by previous reviewers.

Reply:
<img width="1016" alt="Screenshot 2023-06-09 at 18 22 05" src="https://github.com/chepkok3/Budget-app/assets/78428936/56d502f5-bada-4af2-a36a-28c1bc10382f">